src/bits.hpp

```diff
@@ -58,6 +58,10 @@ struct SmallVector {
         return count_;
     }
 
+    void resize(const size_type n) {
```

Comment:
Also are these values zero'd out? I'd be wary of just increasing the size.

Reply:
This should work for LargeVector since `std::vector` has `resize()` method with a similar prototype.

Zeroing is not necessary since in functions places where `resize()` is used, `values_[]` get assigned as well.